pyroomacoustics/test_pra.py

```diff
@@ -0,0 +1,591 @@
+import matplotlib
+
```

Comment:
The test files should be located in `pyroomacoustics/tests` and should be able to run automatically. Is it the case ?

They should be functions with a name that starts with `test_`, e.g., `test_dirpat_stuff`, take no argument, and return nothing. The test success/failure is determined using assert.

Typically, you want to have some assert fail if the code does not work as expected. You can check some of the files in `tests` to see how that works.